waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java

```diff
@@ -671,4 +672,24 @@ public void metastoreTunnelConfiguration() throws Exception {
     assertThat(metastoreTunnel.getPrivateKeys(), is(privateKeys));
   }
 
+  @Test
+  public void getDatabaseFromPatternManual() throws Exception {
```

Comment:
Should I also add a test for getting database from pattern using PREFIXED database resolution?

Reply:
that would be great!